ci/bin/format.dart

```diff
@@ -106,17 +104,17 @@ Future<String> _runGit(
   return result.stdout;
 }
 
-typedef MessageCallback = Function(String message, {MessageType type});
+typedef MessageCallback = Function(String? message, {MessageType type});
 
 /// Base class for format checkers.
 ///
 /// Provides services that all format checkers need.
 abstract class FormatChecker {
   FormatChecker({
-    ProcessManager /*?*/ processManager,
-    @required this.baseGitRef,
-    @required this.repoDir,
-    @required this.srcDir,
+    ProcessManager? processManager,
```

Comment:
Could we default here instead of in the initialiser list?

Reply:
Yes, that is more idiomatic. I did it the other way before because passing on an argument you received from another constructor might send in an unexpected explicit null. But with null safety we don't need to do that anymore! Yay!